tachyon/zk/plonk/circuit/examples/simple_circuit.h

```diff
@@ -158,7 +158,7 @@ class FieldChip {
   }
 
   void ExposePublic(Layouter<F>* layouter, const AssignedCell<F>& cell,
-                    size_t row) const {
+                    RowIndex row) const {
```

Comment:
Could you please import row_index.h for `RowIndex`?

Reply:
I intend to not include any of header other than `circuit.h`. For user's convenience, I want this header to be the only header to be included.